src/getConsentFor.ts

```diff
@@ -46,10 +46,11 @@ export const getConsentFor = (
 	}
 
 	if (consent.aus) {
-		if (typeof consent.aus.rejectedVendors === 'undefined') return true;
-		const rejected = consent.aus.rejectedVendors.filter(
+		if (typeof consent.aus.rejectedCategories === 'undefined') return true;
+		const advertising = '5f859c3420e4ec3e476c7006';
```

Comment:
sorry for the post-release comment. just had a small refactoring idea, if we can move the `advertising` const as a new key into the `VendorIDs` and then we can access it directly with `VendorIDs.advertising

Reply:
I did think that too, actually. We should call it 'aus-advertising'

Reply:
#353